### ISSUE TITLE
minecraft whitelist

### ISSUE THREAD

**Issue by jamesgeddes** [GeekZoneHQ/web#56]
Add member minecraft username, see #55, to the Geek.Zone/Minecraft whitelist. If member leaves Geek.Zone, remove member from Geek.Zone/Minecraft whitelist.

**Comment by CarwynNelson** [comment]
Being a pedant and changing this to allowlist. More inclusive language and all that :)

**Comment by jamesgeddes** [comment]
Not my language mate lol

https://minecraft.gamepedia.com/Commands/whitelist

**Comment by CarwynNelson** [comment]
Yeah I know that's the terminology the minecraft server uses, but eh.

**Comment by jamesgeddes** [comment]
Could it get confusing if we have [two terms for the same thing](http://news.bbc.co.uk/1/hi/sci/tech/686674.stm)? I get the accessibility sentiment, however it appears to be a [widely used term](https://en.wikipedia.org/wiki/Whitelisting).

**Comment by jamesgeddes** [comment]
Just to be clear, I don't mind what we call it as long as it works lol

**Comment by CarwynNelson** [comment]
> Could it get confusing if we have two terms for the same thing?

In general yeah I think it could - but I think most people know that these are synonymous terms

>  it appears to be a widely used term.

It does, but IMHO allowlist and denylist/blocklist are clearer terms and just because something is often done doesn't mean it's the best way to do it.

I feel like I've opened a bit of a can of worms here. At work we are trying to reduce our usage of terms like master/slave whitelist/blacklist etc and so I've tried to carry that over to stuff that I work on in my personal time is all.

**Comment by jamesgeddes** [comment]
You are right, one should always strive for better. Let's go for the more accurate terms.

**Comment by jamesgeddes** [comment]
Just to be clear, I am **not** trying to argue against inclusiveness! Sorry if that's what came across.

**Comment by CarwynNelson** [comment]
> Just to be clear, I am **not** trying to argue against inclusiveness! Sorry if that's what came across.

Not at all
